### PR TITLE
Build processes based on exported data

### PR DESCRIPTION
**Pull request by sethrj**
The loop demo app now loads processes based on what's in the ROOT file (or what's in the GeantImporter-loaded data).

### CODE REVIEW

**Comment by stognini** [comment]
Great work. It would be nice to have a unit test for the ProcessBuilder, but I know we are in a time crunch. Nevertheless, my couple tests here seemed to run okay building the processes. So if you think we're fine for now without the test, for the changes look good and we can merge this in.

**Review by stognini — APPROVED** [review]

**Comment by sethrj** [comment]
Oh yeah, I was going to wait until the previous branch got merged, then add to the GeantSetup test. I can do that or wait until another follow-on PR for improving our ability to set up geant through celeritas for testing...

**Pull request merged**
